tests/tests_full_scan_engine_udf.py

```diff
@@ -565,10 +566,20 @@ async def test_udf_postgresql_and_mysql(self):
         gt_res = sorted(gt_res)
         aidb_res = sorted(aidb_res)
 
+        relative_diffs = []
         for i in range(len(gt_res)):
+          relative_diff = []
           for element1, element2 in zip(gt_res[i], aidb_res[i]):
-            if isinstance(element1, (int, float, Decimal)):
-              assert abs(int(element1) - int(element2)) <= 1
+            if isinstance(element1, (int, float, Decimal)) and (element2 != 0 or element1 !=0):
+              x = (2 * abs(element1 - element2)) / (abs(element1) + abs(element2)) * 100
```

Comment:
Why do not use 'abs(gt_res[i] - aidb_res[i]) / gt_res[i]'?

Reply:
we get errors in case gt_res[i] is zero. So, used the mean of both results